models/meshery_pattern_persister.go

```diff
@@ -25,6 +25,8 @@ type MesheryPatternPage struct {
 
 // GetMesheryPatterns returns all of the performance profiles
 func (mpp *MesheryPatternPersister) GetMesheryPatterns(search, order string, page, pageSize uint64) ([]byte, error) {
+	order = sanitizeOrderInput(order, []string{"created_at", "updated_at", "name"})
```

Comment:
How do we make this new function compulsory for all Get() functions that invoke SQL queries?

Do we need to consider Update() and Delete() functions as receiving the sanitizeOrderInput() as well?

Reply:
This function is now invoked before every `SELECT` type query where we expose a portion of the query in the API. I am not sure if can get any more stricter here as the function needs to know the "whitelisted" column names and they differ for each table.

We won't need the custom sanitization in the functions like `Update` or `Delete` as they don't use any of the SQL injection unsafe method listed here: https://gorm.io/docs/security.html

Reply:
Thank you, @utkarsh-pro :thu